odoo/addons/base/models/res_users.py

```diff
@@ -741,8 +814,8 @@ def write(self, values):
                 self = self.sudo()
 
         old_groups = []
-        if 'groups_id' in values and self._apply_groups_to_existing_employees():
-            # if modify groups_id content, compute the delta of groups to apply
+        if 'group_ids' in values and self._apply_groups_to_existing_employees():  # TODO: remove default user stuff => use a ir.config_parameter default group (with implied groups)
```

Comment:
Maybe we can deprecate/remove this feature?

Reply:
This will be removed to use a default group in a future pr